test/test_torch.py

```diff
@@ -2496,6 +2496,247 @@ def test_index(self):
         self.assertRaises(TypeError, lambda: reference[0.0, ..., 0.0:2.0])
         self.assertRaises(TypeError, lambda: reference[0.0, :, 0.0])
 
+    @staticmethod
+    def _test_advancedindex(self, conv_fn):
+        # Tests for Integer Array Indexing, Part I - Purely integer array
+        # indexing
+
+        def consec(size, start=1):
+            sequence = torch.ones(int(torch.Tensor(size).prod(0)[0])).cumsum(0)
+            sequence.add_(start - 1)
+            return sequence.resize_(*size)
```

Comment:
Never use `resize_` like that (tensor content after resize is unspecified). This is what `view` is for. It works now, but would silently give incorrect results if the tensor was noncontiguous